src/files.ts

```diff
@@ -129,7 +129,12 @@ export function parseFileIgnores(path: string): string[] {
       );
     }
   }
-  return parseIgnoreRulesToGlobs(rules, dirname);
+  try {
+    return parseIgnoreRulesToGlobs(rules, dirname);
+  } catch (err) {
+    console.error('Could not parse ignore rules to glob', { path, error: (err as Error).message });
+    throw new Error('Ignore rules should be in .gitignore format');
```

Comment:
The problem was occurring in `.snyk` file, which doesn't follow `.gitignore` format.

Reply:
@michelkaporin The problem can actually happen any ignore file. Not .snyk only. This is why I added path to the console error, so end user will be able to see where the problematic rule was defined